pkg/networkmonitor/values.go

```diff
@@ -0,0 +1,4 @@
+package networkmonitor
+
+// WhitelistPKs store whitelisted keys of network monitor
+type WhitelistPKs map[string]struct{}
```

Comment:
We can also move this logic `api.WhitelistPKs[v] = struct{}{}` as a `Set()` func along with this as `_, ok := WhitelistPKs[nmPkString]` `Get()`

Reply:
Done